libs/googletest/Makefile

```
@@ -0,0 +1,73 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=googletest
+PKG_VERSION:=1.17.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/google/googletest/archive/refs/tags/
+PKG_SOURCE_URL_FILE:=v$(PKG_VERSION).tar.gz
+PKG_HASH:=65fab701d9829d38cb77c14acdc431d2108bfdbf8979e40eb8ae567edf10b27c
+
+PKG_BUILD_PARALLEL:=1
+PKG_MAINTAINER:=Jungo Lin <jungolin.tw@gmail.com>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+
```

```suggestion
PKG_CPE_ID:=cpe:/a:google:google_test

```

It would be nice to assign a CPE ID.
https://nvd.nist.gov/products/cpe/search/results?keyword=cpe:2.3:a:google:google_test